### DISCUSSION TITLE
Obico not installed

### DISCUSSION THREAD

**Discussion by Ackta** [Guilouz/Creality-Helper-Script-Wiki#431] [Issues]
Hi when i tru to reinstall obico 
Info: Downloading Moonraker Obico...
Cloning into '/usr/data/moonraker-obico'...
remote: Enumerating objects: 3258, done.
remote: Counting objects: 100% (1389/1389), done.
remote: Compressing objects: 100% (341/341), done.
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: the remote end hung up unexpectedlyiB | 111.00 KiB/s
fatal: early EOF
fatal: index-pack failed

**Comment by Guilouz** [comment]
This is a download issue, not a script issue
